_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-cayman
```

Comment:
Why do we need to set this theme? I found that's for github pages built from markdown sources but we use CRA + bootstrap

Reply:
Yes, was testing things and file slipped in, not necessary